pkg/controller/secret/secret_controller.go

```diff
@@ -465,12 +479,13 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 	reqLogger := log.WithValues("Request.Name", request.Name)
 	reqLogger.Info("Reconciling Secret")
 
-	// This operator is only interested in the 3 secrets listed below. Skip reconciling for all other secrets.
+	// This operator is only interested in the 3 secrets & 1 configMap listed below. Skip reconciling for all other objects.
```

Comment:
Minor nit, but with the watch of confimaps, 480 should also be updated

Reply:
Changed `"Reconciling Secret"` -> `"Reconciling Object"`